src/plugins/kibana/public/dashboard/index.html

```diff
@@ -1,5 +1,56 @@
 <div dashboard-app class="app-container dashboard-container">
-  <navbar ng-show="chrome.getVisible()" name="dashboard">
+  <navbar name="dashboard-options">
```

Comment:
Can we use this change as an opportunity to move this whole block into a standalone component directive?